src/rubrix/client/rubrix_client.py

```diff
@@ -267,7 +267,7 @@ def load(
         _check_response_errors(response)
 
         records_sorted_by_id = sorted(
-            [record.to_client() for record in response.parsed], key=lambda x: x.id
+            [record.to_client() for record in response.parsed], key=lambda x: str(x.id)
```

Comment:
This change penalizes sorts with homogeneous ids. (Sorting integers [1,2,3,11] by string representation results in a wrong sort)

Could we just avoid ordering in these cases?

Reply:
:+1: I added a try-except block that first tries to sort by int if possible, if not sorts all by str.

My initial motivation of sorting was actually not the absolute order, but that all `rb.load` calls returned the records in the same relative order. But if we can provide a nice int order of the ids (when possible), even better.

Reply:
great !